pkg/appgw/backendhttpsettings.go

```diff
@@ -44,10 +44,18 @@ func (c *appGwConfigBuilder) BackendHTTPSettingsCollection(cbCtx *ConfigBuilderC
 		// as a managed rule would be overwritten.
 		agicHTTPSettings = brownfield.MergeHTTPSettings(allExistingSettings, agicHTTPSettings)
 	}
+	if cbCtx.EnableIstioIntegration {
+		istioHTTPSettings, _, _, _ := c.getIstioDestinationsAndSettingsMap(cbCtx)
+		if istioHTTPSettings != nil {
+			sort.Sort(sorter.BySettingsName(istioHTTPSettings))
+		}
+		agicHTTPSettings = append(agicHTTPSettings, istioHTTPSettings...)
```

Comment:
I was wondering what the semantics are of `istioHTTPSettings...` in the case of `istioHTTPSettings` being `nil`
In retrospect - `append works on nil slices` -- https://play.golang.org/p/UbNsJihcmnM +  https://tour.golang.org/moretypes/15